corehq/apps/analytics/tasks.py

```diff
@@ -63,3 +64,19 @@ def track_built_app_on_hubspot(webuser):
     if vid:
         # Only track the property if the contact already exists.
         _track_on_hubspot(webuser, {'built_app': True})
+
+
+@task(queue='background_queue', acks_late=True, ignore_result=True)
+def track_usage(email, event, properties=None):
```

Comment:
non-blocking, but might want to use a function name that implies this is using kissmetrics